Duo/openEuler/README.md

```diff
@@ -185,63 +186,115 @@ Screen recording (skipping compilation, from image creation to system login):
+[  OK  ] Finished Permit User Sessions.
+[  OK  ] Started Deferred execution scheduler.
+[  OK  ] Started Command Scheduler.
+         Starting Hold until boot process finishes up...
+         Starting Terminate Plymouth Boot Screen...
+[  OK  ] Finished Hold until boot process finishes up.
+[  OK  ] Finished Terminate Plymouth Boot Screen.
+
+openEuler 23.09
+Kernel 5.10.4-tag- on an riscv64
+
+openeuler-riscv64 login: root
+openEuler12#$
+Password: 
+login: timed out after 60[FAILED] Failed to start Dynamic System Tuning Daemon.
+[FAILED] Failed to start firewalld - dynamic firewall daemon.
+[  275.471596] bm-dwmac 4070000.ethernet end0: PHY [stmmac-0:00] driver [Generic PHY] (irq=POLL)
+[  275.564174] dwmac1000: Master AXI performs any burst length
+[  275.612691] bm-dwmac 4070000.ethernet end0: No Safety Features support found
+[  275.658329] bm-dwmac 4070000.ethernet end0: IEEE 1588-2002 Timestamp supported
+[  275.746979] bm-dwmac 4070000.ethernet end0: configuring for phy/rmii link mode
+
+openEuler 23.09
 Kernel 5.10.4-tag- on an riscv64
 
 openeuler-riscv64 login: root
 Password: 
-Last failed login: Thu Jan  1 08:00:58 CST 1970 on ttyS0
-There were 2 failed login attempts since the last successful login.
 
 
 Welcome to 5.10.4-tag-
 
-System information as of time:  Thu Jan  1 08:00:30 CST 1970
+System information as of time:  Mon Sep 18 08:06:49 CST 2023
 
-System load:    1.68
-Processes:      70
-Memory used:    61.2%
+System load:    8.62
+Processes:      67
+Memory used:    79.0%
 Swap used:      0.0%
-Usage On:       3%
+Usage On:       7%
 Users online:   1
 
 
-[root@openeuler-riscv64 ~]# ./neofetch 
-                 `.cc.`                                                                                                         
-             ``.cccccccc..`                ---------------------- 
-          `.cccccccccccccccc.`             OS: openEuler 23.03 riscv64 
-      ``.cccccccccccccccccccccc.``         Host: Milk-V Duo 
-   `..cccccccccccccccccccccccccccc..`      Kernel: 5.10.4-tag- 
-`.ccccccccccccccc/++/ccccccccccccccccc.`   Uptime: 35 secs 
-.cccccccccccccccmNMMNdo+oso+ccccccccccc.   Shell: bash 5.1.9 
-.cccccccccc/++odms+//+mMMMMm/:+syso/cccc   Terminal: /dev/ttyS0 
-.cccccccccyNNMMMs:::/::+o+/:cdMMMMMmcccc   CPU: (1) 
-.ccccccc:+NmdyyhNNmNNNd:ccccc:oyyyo:cccc   Memory: 38MiB / 54MiB 
-.ccc:ohdmMs:cccc+mNMNmyccccccccccccccccc
-.cc/NMMMMMo////:c:///:cccccccccccccccccc                           
-.cc:syysyNMNNNMNyccccccccccccccccccccccc                           
-.cccccccc+MMMMMNyc:/+++/cccccccccccccccc
-.cccccccccohhhs/comMMMMNhccccccccccccccc
-.ccccccccccccccc:MMMMMMMM/cccccccccccccc
-.ccccccccccccccccsNNNNNd+cccccccccccccc.
-`..cccccccccccccccc/+/:cccccccccccccc..`
-   ``.cccccccccccccccccccccccccccc.``
-       `.cccccccccccccccccccccc.`
-          ``.cccccccccccccc.``
-              `.cccccccc.`
-                 `....`
-
+[root@openeuler-riscv64 ~]# ./neofetch
+-bash: ./neofetch: No such file or directory
```

Comment:
`neofetch` is not necessary.
Anyway no big issue here. You can leave it as-is.

Reply:
This is just because @wychlw 's original report contained the use of `neofetch`... Seems that it is previously bundled with openEuler's rootfs